bluesky/run_engine.py

```diff
@@ -228,6 +228,18 @@ def __init__(self, md=None, *, loop=None, preprocessors=None,
         if md is None:
             md = {}
         self.md = md
+        self.md.setdefault('versions', {})
+
+        try:
+            import ophyd
+            self.md['versions']['ophyd'] = ophyd.__version__
+        except ImportError:
+            self.log.debug("Failed to import __version__ from ophyd.")
```

Comment:
I'd be more precise here:
```suggestion
            self.log.debug("Failed to import ophyd.__version__.")
```